k8s/migration/pkg/constants/constants.go

```diff
@@ -72,6 +72,9 @@ const (
 	// PauseMigrationLabel is the label for pausing rolling migration plan
 	PauseMigrationLabel = "vjailbreak.k8s.pf9.io/pause"
 
+	// TrueString is the string representation of boolean true
+	TrueString = "true"
```

Comment:
I think there is already some constant for this, please check and try if you can reuse it

Reply:
https://github.com/platform9/vjailbreak/blob/1b741ce4ea85dfbdf858b2f71c60fb50aa7e2689/k8s/migration/pkg/utils/credutils.go#L54
okay, then please change here also - https://github.com/platform9/vjailbreak/blob/1b741ce4ea85dfbdf858b2f71c60fb50aa7e2689/k8s/migration/pkg/utils/credutils.go#L54